### ISSUE TITLE
Move example outside of allOf in schemas.Duration

### ISSUE THREAD

**Issue by vlasatno** [camaraproject/QualityOnDemand#282]
**Describe the bug**
The Duration schema has an example that is dashed such that it is an element in an allOf list, whereas it should be a member of the $ref above it.

```yaml
    Duration:
      type: object
      properties:
        value:
          type: integer
          example: 12
          format: int32
          minimum: 1
        unit:
          allOf:
            - $ref: "#/components/schemas/TimeUnitEnum"
            - example: Minutes
```

Should become:
```yaml
    Duration:
      type: object
      properties:
        value:
          type: integer
          example: 12
          format: int32
          minimum: 1
        unit:
          allOf:
            - $ref: "#/components/schemas/TimeUnitEnum"
              example: Minutes
```

**To Reproduce**
Not applicable.

**Expected behavior**
Not applicable.

**Screenshots**
Not applicable.

**Environment:**
Not applicable.

**Additional context**
Found when generating `python-fastapi` code with [openapi-generator](https://github.com/OpenAPITools/openapi-generator)

**Comment by jlurien** [comment]
The current spec is weird and some generators may have problems with it, but the proposed fix is also wrong.

We should probably change it as

```
    Duration:
      type: object
      properties:
        value:
          type: integer
          example: 12
          format: int32
          minimum: 1
        unit:
          $ref: "#/components/schemas/TimeUnitEnum"
```

and remove the example or move it to the TimeUnitEnum schema

**Comment by jlurien** [comment]
Fixed as part of #283
